widgetssdk/src/main/java/com/glia/widgets/fcm/PushNotifications.kt

```diff
@@ -32,12 +33,32 @@ interface PushNotifications {
      * @param message - New FCM message
      */
     fun onNewMessage(message: RemoteMessage)
+
+    /**
+     * Should be called when the main activity is created.
+     *
+     * **Usage example:**
+     * ```kotlin
+     * override fun onCreate(savedInstanceState: Bundle?) {
+     *    super.onCreate(savedInstanceState)
+     *
+     *    ...
+     *
+     *    val push = GliaWidgets.getPushNotifications().handleOnMainActivityCreate(intent.extras)
+     *
+     *    if (pushMessage?.type == GliaPushMessage.PushType.CHAT_MESSAGE) {
```

Comment:
`val` name is `push`. Maybe there should be `push` instead of `pushMessage` here. 🤔